testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -165,26 +164,24 @@ public void testInsert()
     }
 
     @Test
-    public void testDelete()
+    public void testRowLevelDelete()
     {
-        if (!hasBehavior(SUPPORTS_DELETE)) {
-            assertQueryFails("DELETE FROM region", "This connector does not support deletes");
```

Comment:
Please bring this back. This check was verifying whether `! hasBehavior(SUPPORTS_DELETE)`  is declared correctly.